Src/AutoFakeItEasy.FakeItEasy5UnitTest/AutoFakeItEasy.FakeItEasy5UnitTest.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="FakeItEasy" Version="[5.0.0]" />
```

Comment:
Differs from other test projects, of course.  I include this change to prove that the fix works with new FakeItEasys. And in general, I think it's better to test with the later versions within a major release, but who has the time to keep these things up to date?
Anyhow, I'm happy to drop this commit, if you like.